docs/reference/docs/update.asciidoc

```diff
@@ -46,8 +46,8 @@ POST test/_doc/1/_update
 // CONSOLE
 // TEST[continued]
 
-We can add a tag to the list of tags (note, if the tag exists, it
-will still add it, since it's a list):
+We can add a tag to the list of tags (note: if the tag exists, it
```

Comment:
The colon seems a bit odd to me as a reader (but then so are three commas). I wonder whether we can solve this dilemma by getting rid of the word "note" alltogether? Is it really helpful here?

Reply:
Agree -- I made the change. Thx!

Reply:
Great! I'll approve the PR then.